drivers/input/input_gt911.c

```diff
@@ -33,6 +32,9 @@ LOG_MODULE_REGISTER(gt911, CONFIG_INPUT_LOG_LEVEL);
 #define REG_CONFIG_SIZE    (186U)
 #define GT911_PRODUCT_ID   (0x00313139U)
 
+static const uint16_t reg_points[] = {BSWAP_16(0x814FU), BSWAP_16(0x8157U), BSWAP_16(0x815FU),
+				      BSWAP_16(0x8167U), BSWAP_16(0x816FU)};
```

Comment:
that's an odd way of doing it, how about using a macro, something like

`#define POINT_ADDR(n) BSWAP_16(POINT_0_ADDR + POINT_OFFSET * n)`

the offset is 8 I suppose, or `sizeof(struct  gt911_point_reg_t)`

Reply:
Indeed, it's better this way, fixed it. Thanks :)